package.json

```diff
@@ -71,13 +71,15 @@
     "webpack-dev-server": "2.11.2"
   },
   "dependencies": {
+    "@patternfly/react-core": "^2.1.4",
     "@rails/webpacker": "^3.5.5",
     "ajv": "^6.9.2",
     "bootstrap-sass": "3",
     "c3": "^0.4.11",
     "classnames": "^2.2.6",
     "core-js": "^2.6.5",
     "decca": "^2.3.0",
+    "emotion": "^9.2.12",
```

Comment:
Is this used anywhere?

Reply:
Is a dependency of patternfly, for any reason is not being installed automatically, I had to install manually

Reply:
It's a dependency of Patternfly already: https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/package.json#L35.
Could you uninstall it, run `npm i` and try again?